app/components/Sidebar/components/DocumentLink.tsx

```diff
@@ -47,7 +47,8 @@ function DocumentLink(
   const { documents, policies } = useStores();
   const { t } = useTranslation();
   const isActiveDocument = activeDocument && activeDocument.id === node.id;
-  const hasChildDocuments = !!node.children.length;
+  const hasChildDocuments =
+    !!node.children.length || activeDocument?.parentDocumentId === node.id;
```

Comment:
Items were not displaying at all when there were no children, but we need to display the list when there's an active draft